src/components/test/ShortcutsModal.spec.js

```diff
@@ -6,15 +6,34 @@ import React from "react";
 import { shallow } from "enzyme";
 import { ShortcutsModal } from "../ShortcutsModal";
 
+function render(overrides = {}) {
+  const props = {
+    enabled: true,
+    handleClose: jest.fn(),
+    ...overrides
+  };
+  const component = shallow(<ShortcutsModal {...props} />);
+
+  return { component, props };
+}
+
 describe("ShortcutsModal", () => {
   it("renders when enabled", () => {
-    const enabled = true;
-    const wrapper = shallow(<ShortcutsModal enabled={enabled} />);
-    expect(wrapper).toMatchSnapshot();
+    const { component } = render();
+    expect(component).toMatchSnapshot();
   });
 
   it("renders nothing when not enabled", () => {
-    const wrapper = shallow(<ShortcutsModal />);
-    expect(wrapper.text()).toBe("");
+    const { component } = render({
+      enabled: false
+    });
+    expect(component.text()).toBe("");
+  });
+
+  it("renders with additional classname", () => {
+    const { component } = render({
+      additionalClass: "additional-class"
+    });
+    expect(component).toMatchSnapshot();
```

Comment:
it would be great if we could just use `.hasClass` here, as it is the thing we are testing for. A snapshot seems to make the test more difficult to read and brittle.

Reply:
Heh, I agree generally when you use normal snapshots because of the size produced but I'm gonna shamelessly plug my tool [`snapshot-diff-serializer`](https://github.com/wldcordeiro/snapshot-diff-serializer) as this is an awesome place for this kind of snapshot.

```js
const { component: defaultComp } = render()
const { component } = render({ additionalClass: "additional-class" })
expect({
  diffA: defaultComp,
  diffB: component
}).toMatchSnapshot();
```

Reply:
fair. `hasSnapshot` is just the lazy option